schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -802,13 +802,21 @@
         "name": {
           "type": "string"
         },
-        "subnet": {
-          "$ref": "#/definitions/id"
-        },
-        "loadBalancerBackendAddressPools": {
-          "type": "array",
-          "items": {
+		"properties" : {
```

Comment:
Did you mean to add this extra layer of properties? This means that ipConfiguration went from matching this:
{
  "name": "blah",
  "subnet": { ... },
  "loadBalancerBackendAddressPools": [ ... ],
}
to
{
  "name": "blah",
  "properties": { ... }
}
In fact, I'm not even sure that this is a valid JSON schema. Do you have any tests written for this definition?